x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -540,7 +540,9 @@ static RoleDescriptor kibanaSystem(String name) {
                         "logs-m365_defender.vulnerability-*",
                         "logs-microsoft_defender_endpoint.vulnerability-*",
                         "logs-microsoft_defender_cloud.assessment-*",
-                        "logs-sentinel_one.application_risk-*"
+                        "logs-sentinel_one.application_risk-*",
+                        "logs-island_browser.user-*",
+                        "logs-island_browser.device-*"
```

Comment:
You don't need this here, because you already adding below in L562 and L563. 
This permissions set is a subset of what you have there.